i18n/pl.toml

```diff
@@ -17,8 +17,10 @@
 [nav_main]
     other = "główna"
 [blog_readingTime]
-    one = "{{ .Count }} minut"
-    other = "{{ .Count }} minut"
+    one = "{{ .Count }} minuta"
+    few = "{{ .Count }} minuty"
+    many = "{{ .Count }} minuty"
```

Comment:
Why do you need extra translations for `few` and `many` if they the same as `other`?

Reply:
Please read the author's 2nd post in the linked GitHub issue https://github.com/gohugoio/hugo/issues/3846 and the 1st reference [Unicode's plural rules](http://www.unicode.org/cldr/charts/28/supplemental/language_plural_rules.html) for different languages (especially Polish).

Please compare the screenshots in my related [Hugo Discourse question](https://discourse.gohugo.io/t/plural-doesnt-work-for-brazillian-portuguese-and-polish/20329).

Reply:
I see. Would you please document, that this is a workaround and targets a bug in hugo.

Reply:
OK for further explanation.  However, that's the expected behavior in Hugo.  Instead, this is a bug in this theme's template.